application/src/main/java/org/thingsboard/server/controller/OtaPackageController.java

```diff
@@ -84,8 +87,9 @@ public class OtaPackageController extends BaseController {
     @PreAuthorize("hasAnyAuthority( 'TENANT_ADMIN')")
     @RequestMapping(value = "/otaPackage/{otaPackageId}/download", method = RequestMethod.GET)
     @ResponseBody
-    public ResponseEntity<org.springframework.core.io.Resource> downloadOtaPackage(@ApiParam(value = OTA_PACKAGE_ID_PARAM_DESCRIPTION)
-                                                                                   @PathVariable(OTA_PACKAGE_ID) String strOtaPackageId) throws ThingsboardException {
+    @Transactional
```

Comment:
Because  _checkOtaPackageId_ method searches for _OtaPackage_, and `Blob `requires using `@Transactional `annotation. Should I change checking method?